docs/Migrating-v3.md

```diff
@@ -0,0 +1,101 @@
+# ECL v3 migration notes
+
+The following guidelines aim to facilitate migration between ECL v2 to v3.
+
+## Deprecated components haven't been migrated
+
+The following have been kept in v2 in order to avoid breaking changes, but will no longer be available in v3:
+
+Components:
+
+- Accordion (deprecated since ECL 2.6.0)
```

Comment:
we may specify that this is the old accordion version (we still have an accordion in ECL 3, which correspond to component accordion2 of ECL 2)